region/lowernorfair/east/Amphitheatre.json

```diff
@@ -702,6 +724,73 @@
       "bypassesDoorShell": true,
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 2],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit) - Outrun the Rising Acid",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        {"obstaclesCleared": ["A"]},
+        {"or": [
+          "h_heatedCrystalFlashForReserveEnergy",
+          {"and": [
+            {"or": [
+              "ScrewAttack",
+              {"and": [
+                "Plasma", "canXRayWaitForIFrames"
+              ]}
+            ]},
+            "h_RModeCanRefillReserves",
+            {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
+          ]}
+        ]},
+        {"enemyKill": {"enemies": [["Yellow Space Pirate (standing)"]]}},
+        {"canShineCharge": {"usedTiles": 20, "gentleDownTiles": 6, "openEnd": 1}},
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt"
+      ],
+      "resetsObstacles": ["R-Mode", "A"],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "If you're fast, you can find enough time to kill a Yellow Pirate with a charged Microwave.",
+        "Kill the pirate at the top, get the shinecharge, and get to one of the other platform pirates",
```

Comment:
The top-left pirate wouldn't shoot at me.  And then my shinespark timer ran out...

Reply:
Environment damage is better so it now looks a lot like the other door's strat, but I'm tentatively putting `heatProof` on using acid, unless there's a good way to represent having to wait for the acid to finish rising.